package.json

```diff
@@ -1,6 +1,37 @@
 {
     "name": "vscode-cfn-lint",
-    "description": "CloudFormation linter",
+    "displayName": "CloudFormation Linter",
+    "description": "AWS CloudFormation Linter",
```

Comment:
It says [here](https://code.visualstudio.com/api/references/extension-manifest#marketplace-presentation-tips):  "Provide a good display name and description. This is important for the Marketplace and in product displays. These strings are also used for text search in VS Code and having relevant keywords will help a lot."

Maybe put something more explicit, like "Linter for AWS Cloudformation templates, both JSON and YAML"?